Otherworldly Pines/Assets/Scenes/Demo Level.unity

```diff
@@ -6070,5 +6072,13 @@ PrefabInstance:
       propertyPath: pauseDuration
       value: 0.2
       objectReference: {fileID: 0}
+    - target: {fileID: 8896395635190139466, guid: c084183a77fd74e4ea6bd9134d6c0ce3, type: 3}
```

Comment:
This is showing up because this instance of the prefab has overrides over the default one, if they're important you should just apply them to the prefab. Otherwise, might be better to remove them for now to avoid merge conflicts